app/Resources/views/subscription/tabs/tab-general.html.twig

```diff
@@ -46,7 +46,12 @@
 
     <div class="row">
         <div class="col-xs-11">
-            {{ form_row(form.contact.email) }}
+            <div class="form-group">
+                <label class="col-sm-2 control-label">{{ 'email'|humanize|trans }}</label>
```

Comment:
Ah ja, dit is een copy van de standaard label template. In dit geval zal er idd niet zo veel spannends gebeuren.